run.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+import csv
+from pathlib import Path
+import os
+import pickle
+import yaml
+
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer
+from bertopic.vectorizers import ClassTfidfTransformer
+from spacy.lang.en import stop_words
+from sentence_transformers import SentenceTransformer
+
+from PatientX.models.BERTopicModel import BERTopicModel
+from PatientX.MistralRepresentation import MistralRepresentation
+
+
+def read_data(datafolder: Path):
+    """
+    Read in data
```

Comment:
Improve this comment. It looks like this is returning a list of \<something>, but I'm not sure what \<something> is.